src/amber/router/cookies/abstract_store.cr

```diff
@@ -0,0 +1,20 @@
+module Amber::Router::Cookies
+  abstract class AbstractStore
+    getter store : Store
+
+    def initialize(@store)
+    end
+
+    def [](name)
+      get(name)
+    end
+
+    def []=(name, value)
+      set(name, value)
+    end
+
+    abstract def get(name)
```

Comment:
I'd use this opportunity to change it to `get?`. This way we have a free `#[]?` and a cheaper way to handle non-existent keys.

Reply:
If possible, I'd rather tackle that sort of improvement in another PR. The focus of this one was to break up the classes, remove dead code and have it all still work.